fix(curriculum/scientific-computing-with-python): improve cipher project (#53069)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/65553159615a8123b190ee43.md

---
id: 65553159615a8123b190ee43
title: Step 86
challengeType: 20
dashedName: step-86
---

# --description--

It works! Now, you are going to start with an encrypted message to be decrypted.

> Change your `text` variable into the string `mrttaqrhknsw ih puggrur`.

Reviewer: ```suggestion
Change the string value of `text` to `mrttaqrhknsw ih puggrur`.
```
